pkg/ebpf/c/tracee.bpf.c

```diff
@@ -2830,6 +2831,84 @@ int tracepoint__sched__sched_process_exit(struct bpf_raw_tracepoint_args *ctx)
     return 0;
 }
 
+SEC("raw_tracepoint/syscall__accept4")
+int syscall__accept4(void *ctx)
+{
+    event_data_t data = {};
+    if (!init_event_data(&data, ctx))
+        return 0;
+
+    args_t saved_args;
+    if (load_args(&saved_args, SOCKET_ACCEPT) != 0) {
+        // missed entry or not traced
+        return 0;
+    }
```

Comment:
Better put this first to save initializing event data if we shouldn't trace (arguments don't exists in map)